hs_core/hydroshare/resource.py

```diff
@@ -1051,7 +1051,11 @@ def submit_resource_for_review(request, pk):
     resource.raccess.save()
 
     # create review date -- must be after review_pending = True
-    resource.metadata.create_element('date', type='review_started', start_date=resource.updated)
+    date = resource.metadata.dates.all().filter(type='review_started').first()
+    if date:
+        resource.metadata.update_element('date', date.id)
```

Comment:
For the updating the date, you have to provide the 'start_date' value.  
Alternatively this may be better:
```
resource.metadata.dates.all().filter(type='review_started').delete()
resource.metadata.create_element('date', type='review_started', start_date=resource.updated)
```

Reply:
I am thinking setting `review_started` to` resource.updated` may cause problem when sending review reminders. It would be better to set to the current date.

Reply:
Good call, thank you Pabitra!